go.mod

```
@@ -6,85 +6,93 @@ require (
+// Pinned to kubernetes-1.17.4
 replace (
-	k8s.io/api => k8s.io/api v0.0.0-20191016110408-35e52d86657a
-	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.0.0-20191016113550-5357c4baaf65
-	k8s.io/apimachinery => k8s.io/apimachinery v0.0.0-20191004115801-a2eda9f80ab8
-	k8s.io/apiserver => k8s.io/apiserver v0.0.0-20191016112112-5190913f932d
-	k8s.io/cli-runtime => k8s.io/cli-runtime v0.0.0-20191016114015-74ad18325ed5
+	k8s.io/api => k8s.io/api v0.17.4
+	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.17.4
+	k8s.io/apimachinery => k8s.io/apimachinery v0.17.4
+	k8s.io/apiserver => k8s.io/apiserver v0.17.4
+	k8s.io/cli-runtime => k8s.io/cli-runtime v0.17.4
 	k8s.io/client-go => k8s.io/client-go v0.0.0-20191016111102-bec269661e48
-	k8s.io/cloud-provider => k8s.io/cloud-provider v0.0.0-20191016115326-20453efc2458
-	k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.0.0-20191016115129-c07a134afb42
-	k8s.io/code-generator => k8s.io/code-generator v0.0.0-20191004115455-8e001e5d1894
-	k8s.io/component-base => k8s.io/component-base v0.0.0-20191016111319-039242c015a9
-	k8s.io/cri-api => k8s.io/cri-api v0.0.0-20190828162817-608eb1dad4ac
-	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.0.0-20191016115521-756ffa5af0bd
-	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.0.0-20191016112429-9587704a8ad4
-	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.0.0-20191016114939-2b2b218dc1df
-	k8s.io/kube-proxy => k8s.io/kube-proxy v0.0.0-20191016114407-2e83b6f20229
-	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.0.0-20191016114748-65049c67a58b
-	k8s.io/kubectl => k8s.io/kubectl v0.0.0-20191016120415-2ed914427d51
-	k8s.io/kubelet => k8s.io/kubelet v0.0.0-20191016114556-7841ed97f1b2
-	k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.0.0-20191016115753-cf0698c3a16b
-	k8s.io/metrics => k8s.io/metrics v0.0.0-20191016113814-3b1a734dba6e
-	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.0.0-20191016112829-06bb3c9d77c9
+	k8s.io/cloud-provider => k8s.io/cloud-provider v0.17.4
+	k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.17.4
+	k8s.io/code-generator => k8s.io/code-generator v0.17.4
+	k8s.io/component-base => k8s.io/component-base v0.17.4
+	k8s.io/cri-api => k8s.io/cri-api v0.17.4
+	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.17.4
+	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.17.4
+	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.17.4
+	k8s.io/kube-proxy => k8s.io/kube-proxy v0.17.4
+	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.17.4
+	k8s.io/kubectl => k8s.io/kubectl v0.17.4
+	k8s.io/kubelet => k8s.io/kubelet v0.17.4
+	k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.17.4
+	k8s.io/metrics => k8s.io/metrics v0.17.4
+	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.17.4
 )
 
 replace github.com/docker/docker => github.com/moby/moby v0.7.3-0.20190826074503-38ab9da00309 // Required by Helm
```

I don't think this `replace` is required after the Helm bump to 3.1. But I think this one is now:
```suggestion
replace github.com/docker/distribution => github.com/docker/distribution v0.0.0-20191216044856-a8371794149d // Required by Helm
```

---

^ same. https://github.com/operator-framework/operator-sdk/pull/2702#discussion_r399804371